src/rt-app.c

```diff
@@ -759,6 +772,15 @@ void *thread_body(void *arg)
 				NULL);
 	}
 
+	/* TODO find a better way to handle that constraint:
+	 * Set the task to SCHED_DEADLINE as far as possible touching its
+	 * budget as little as possible for the first iteration.
+	 */
+
+	/* Set scheduling policy and print pretty info on stdout */
+	log_notice("[%d] Starting with %s policy with priority %d", data->ind, policy_to_string(data->sched_data->policy), data->sched_data->prio);
+	set_thread_priority(data, data->sched_data);
```

Comment:
OK. But we seem to do the right thing for set_thread_affinity.
I wonder if we can take inspiration from that.